agent/src/main/java/com/code_intelligence/jazzer/api/Jazzer.java

```diff
@@ -491,6 +501,18 @@ public static void reportFindingFromHook(Throwable finding) {
     }
   }
 
+  private static int getLibFuzzerSeed() {
+    // The Jazzer driver sets this property based on the value of libFuzzer's -seed command-line
+    // option, which allows for fully reproducible fuzzing runs if set. If not running in the
+    // context of the driver, fall back to a random number instead.
+    String rawSeed = System.getProperty("jazzer.seed");
+    if (rawSeed == null) {
+      return new SecureRandom().nextInt();
+    }
+    // If jazzer.seed is set, we expect it to be a valid integer.
+    return Integer.parseUnsignedInt(rawSeed);
```

Comment:
@bertschneider and I are debugging a weird issue where `rawSeed` is not null, but the empty string if this method is called early (e.g. during the agent's `premain`).

Reply:
Fixed, it was just a bug in the C++ part combined with very lazy class loading masking the root cause.